firestore-bigquery-export/guides/GENERATE_SCHEMA_VIEWS.md

```diff
@@ -314,6 +314,62 @@ Since all document data is stored in the schemaless changelog, mistakes in
 schema configuration don't affect the underlying data and can be resolved by
 re-running the schema-views script against an updated schema file.
 
+### Using Gemini to generate draft schema files
+
+Instead of manually creating schema files, you can use the built-in Gemini AI Agent to automatically analyze your Firestore collection and generate an appropriate schema. The agent will:
+
+1. Sample documents from your collection
+2. Analyze the data structure
+3. Generate a well-documented schema file
+4. Create the corresponding BigQuery views
+
+You can use Gemini in either interactive or non-interactive mode:
+
+```bash
+# Interactive mode
+npx @firebaseextensions/fs-bq-schema-views
```

Comment:
I think its better to include a copy-paste minimal usage here, e.g. add `--use-gemini-agent` and other required params

Reply:
We have a non-interactive example below, can you explain what you mean? confused